src/interface/blas3/backend/amd_gpu.hpp

```diff
@@ -30,8 +30,8 @@ namespace blas {
 namespace gemm {
 
 namespace backend {
-template <bool _t_a, bool _t_b, bool is_beta_zero, typename sb_handle_t,
-          typename container_0_t, typename container_1_t,
+template <bool _t_a, bool _t_b, bool s_a, bool s_b, bool is_beta_zero,
```

Comment:
This is just add `s_a` and `s_b` to all the template `Gemm_Launcher` templates on the backend folder.